docs/getting_started/basic_usage/data.rst

```diff
@@ -0,0 +1,55 @@
+.. _data:
+
+=================
+Data Input/Output
+=================
+
+The parameters that are used to correctly define the data inputs are the three dataset variables ``train``, ``dev`` and
```

Comment:
Make newlines at sensible places (e.g. after a sentence, or after a logical break).
It should be such that further edits lead to minimal diffs.